src/components/Cards/EarnCard/variants/CompactEarnCard.tsx

```diff
@@ -42,7 +43,7 @@ export const CompactEarnCard: FC<Omit<EarnCardProps, 'variant'>> = ({
   const items = useMemo(() => {
     const result = [];
 
-    if (apy) {
+    if (apy && apy.total) {
```

Comment:
You're going to skip the case where apy.total is 0 on purpose?

Reply:
Fair point 🙌 eagle eyes